appveyor.yml

```diff
@@ -1,3 +1,4 @@
+image: Visual Studio 2019
```

Comment:
I think this part [breaks Python 2.7 on Appveyor](https://ci.appveyor.com/project/mgedmin/zope-interface/builds/36147882/job/u74jepxgii99ix5k):

```
********************************************************************************
WARNING:
        An optional code optimization (C extension) could not be compiled.
        Optimizations for this package will not be available!
Microsoft Visual C++ 9.0 is required. Get it from http://aka.ms/vcpython27
********************************************************************************
```

Reply:
Let's try `Visual Studio 2015` (aka the default) even though it does not seem to have Python 3.9.

Reply:
Our appveyor.yml has a powershell snippet that can install missing Python versions using the official installers from python.org.

Reply:
Sorry, I always forget this.